src/test/java/uk/gov/hmcts/reform/ccd/document/am/service/impl/DocumentManagementServiceImplTest.java

```diff
@@ -456,6 +456,15 @@ void checkServicePermissionForUpload_WhenJurisdictionIdIsNull() {
         assertEquals(Boolean.FALSE, result);
     }
 
+    @Test
+    void checkServicePermission_WhenServiceIdIsNotAuthorised() {
+        when(securityUtilsMock.getServiceId()).thenReturn("ccd_case_document_am_api");
```

Comment:
Maybe we can pass a dummy service which doesn't looks legitimate, like "bad_Service_name".